app/configurations/config.turku.js

```diff
@@ -203,4 +233,23 @@ export default configMerger(walttiConfig, {
 
   showNearYouButtons: true,
   allowLogin: false,
+  constantOperationStops: {
+    'FOLI:9900': CONSTANT_OPERATION_PARAGRAPHS.fori,
+    'FOLI:9901': CONSTANT_OPERATION_PARAGRAPHS.fori,
+    'FUNI:9902': CONSTANT_OPERATION_PARAGRAPHS.funi,
+    'FUNI:9903': CONSTANT_OPERATION_PARAGRAPHS.funi,
+  },
+  constantOperationRoutes: {
+    'FOLI:180': CONSTANT_OPERATION_PARAGRAPHS.fori,
+    'FUNI:2': CONSTANT_OPERATION_PARAGRAPHS.funi,
+  },
+  strongWalkReluctance: {
+    weights: {
+      FUNICULAR: 0.25,
+    },
+    walkReluctance: 15,
```

Comment:
I think having this high walkReluctance value will slow down the routing by quite a bit. Could the same effect be achieved by lowering the funicular weight even more?